cli_reference/get_started_cli.adoc

```diff
@@ -53,6 +53,12 @@ endif::[]
 
 Installation options for the CLI vary depending on your operating system.
 
+To log in using the CLI, collect your token from the web console's *Command
+Line* page, which is accessed from *Command Line Tools* in the *Help* menu. The
+token is hidden, so you must click the *copy to clipboard* button at the end of
+the `oc login` line on the *About* page, then paste the copied contents to show
```

Comment:
This '*About* page' should be updated, too.

Reply:
Thanks! Updated.